programs/bpf_loader/src/lib.rs

```diff
@@ -162,12 +163,45 @@ pub fn process_instruction(
     instruction_data: &[u8],
     invoke_context: &mut dyn InvokeContext,
 ) -> Result<(), InstructionError> {
-    solana_logger::setup_with_default("solana=info");
-
+    process_instruction2(
```

Comment:
Doing these gymnastic tricks to get make the lifetime's work when sharing` InvokeContext` in an `Rc<RefCell>>`.  Please help!